index.rst

```diff
@@ -0,0 +1,190 @@
+
+  .. figure:: /_static/figure_2.png
+     :name: figure_2
+     :target: _images/figure_2.png
+
+     Comparison of shear bias with different stamp sizes
+
+One  question frequently asked is how this relates to the LSST requirements for achieving a particular accuracy in shear measurement.  The answer is that we have not attempted to calibrate the shear algorithms tested in this study, so our comparisons are only intended to show relative differences between different measurement algorithms.  A further calibration study is probably needed to say more. This study also did not include spatial variation in the Psf, which would be needed for an accurate assessment.
+
+How errors were determined:
+---------------------------
+
+In the tests discussed below, we really ran only two simulations.  One was the set of 200 subfields, each subfield having 10000 galaxies with the same applied shear.  The other was a set of 6x1024 subfields at 6 values of total shear, with 1024 galaxies at each rotation angle.
+
+Looking forward, I would like to run more random simulations which give us a better idea of how our shear biases vary from simulation to simulation.
+
+We calculated our errors in two ways, one using estimators of the errors from each subfield to analytically determine the errors in the slope and intercept measurements. The other was to use a bootstrap technique to see how the bias varied within resampled distributions.
+
+1.  The analytic approach
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If (e1,e2) is the measure of the galaxy ellipticity and (g1,g2) is the applied shear, the deviation of (e1-g1,e2-g2) from (0,0) is a measure of our error.  Of course, this measurement is dominated by shape noise. By averaging e1 and e2 over and entire subfield, we get an estimator of the applied shear, as well as the error in this estimation.
+
+Since the regression to determine m1, c1, m2 and c2 uses as its data the (e1avg, e2avg)  vs. (g1, g2), we can use the errors in e1avg and e2avg to determine the errors in the regression values.  See Numerical Recipes section 15.2 for a description of this calculation.
+
+2.  Bootstrap estimation
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+However, since we did not entirely trust the analytic approach, we attempted to determine the error in our shear bias calculation by running our entire simulation 100-300 times using a bootstrap technique.  That is, we did each subfield measurement again (for each of the 200 subfields) with 5000 galaxy pairs, sampled from the original 5000 galaxy pairs with replacement.  The bootstrap method consistently gives an error which is larger than the analytic approach by a factor of 2.1-2.3.
```

Comment:
I'm not sure I follow what you mean by "pairs" here, or why 5000 rather than 10000. DM-5357 reckons that you need 250 iterations of resampling to converge -- why do you say "100-300" rather than 250?